go/api/response_handlers.go

```diff
@@ -321,6 +321,29 @@ func handleIntArrayResponse(response *C.struct_CommandResponse) ([]int64, error)
 	return slice, nil
 }
 
+func handleIntOrNilArrayResponse(response *C.struct_CommandResponse) ([]int64, error) {
```

Comment:
We already have https://github.com/niharikabhavaraju/valkey-glide/blob/34bd8722ad34a0ba7b37407a5752c8851feee9ce/go/api/response_handlers.go#L305

Or you want `nil`lable ints there?
```suggestion
func handleIntOrNilArrayResponse(response *C.struct_CommandResponse) ([]Result[int64], error) {
```

Reply:
done